ovs/v1/ovs.yaml

```diff
@@ -223,13 +160,126 @@ paths:
             application/json:
               schema:
-                $ref: '#/components/schemas/transportCall'
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/transportCall'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/error'
+    delete:
+      tags:
+        - Operational Vessel Schedules
+      summary: Delete a transport-call by its transportCallId
+      parameters:
+        - $ref: '#/components/parameters/transportCallID'
+      responses:
+        '204':
+          description: Subscription stopped
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/error'
+    put:
+      tags:
+      - Operational Vessel Schedules
+      summary: Alter a TransportCall
+      parameters:
+        - $ref: '#/components/parameters/transportCallID' 
+      requestBody:
+        description: Parameters used to configure the subscription
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/transportCall'
+      responses:
+        '200':
+          description: Transportcall updated
+          content:
+            application/json:
+              schema:
+               $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/transportCall'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/error'
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/error'
+                
+  /transport-calls/{transportCallID}/transport-events:
+    get:
+      tags:
+        - Operational Vessel Schedules
+      summary: Find timestamps by transport-callID
+      description: Returns all timestamps by a transport-callID
+      parameters:
+        - $ref: '#/components/parameters/transportCallID'
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/transportEvent'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/error'
```

Comment:
same

Reply:
Good point, I will discuss this, let us use the global one for now